src/coreclr/gcinfo/gcinfodumper.cpp

```diff
@@ -465,6 +472,9 @@ PORTABILITY_ASSERT("GcInfoDumper::ReportPointerRecord is not implemented on this
                     assert(!"unimplemented on LOONGARCH yet");
                     //TODO: should confirm ?
                     base = GC_SP_REL;
+#elif defined(TARGET_RISCV64)
+                    assert(!"unimplemented on RISCV64 yet");
+                    base = GC_SP_REL;
```

Comment:
Note, this is unimplemented block (and will not be implemented in this PR for sure). By default, I use same code here as arm/loongarch, since we will have same logic as arch with volatile registers, but could switch to default from `#else` block for sure.